src/utils/index.ts

```diff
@@ -23,7 +22,6 @@ const ShopifyUtils = {
   safeCompare,
   storeSession,
   validateHmac,
-  validateShop,
```

Comment:
I think this all makes sense, but if we stop exporting the function from here, this becomes a breaking change because it changes the lib's external API. We could probably keep the function around and `console.warn` until the next major release, just so we don't tack on a major release right after the last one?

Reply:
Good call, I brought the function + test back and added the deprecation notice.